public/js/checkin.js

```diff
@@ -17,5 +17,15 @@ window.HLRDESK.init.checkin = function() {
   
   $("#checked-out-items tbody tr").click(function(){
     $( this ).toggleClass( "selected" );
+    var checkInButton = document.querySelector('.check-in-btn')
+    var selected = document.querySelector('.selected');
+
+  if(selected.length === 0) {
```

Comment:
The issue with this code is that selected will never have a `length`, because `document.querySelector` only returns one element, rather than multiple elements. To fix this, you can do one of two things:

Either 1) change `document.querySelector('.check-in-btn')` to `document.querySelectorAll('.check-in-btn')`

or 2) change `if(selected.length === 0)` to `if(selected === null)`